diff --git example/geant4/trackingmanager-offload.cc example/geant4/trackingmanager-offload.cc
index b57c9400a..b17b0e783 100644
--- example/geant4/trackingmanager-offload.cc
+++ example/geant4/trackingmanager-offload.cc
@@ -33,8 +33,13 @@
 #endif
 
+// Celeritas
+#include <CeleritasG4.hh>
+
 // Celeritas convenience utils
 #include <corecel/Assert.hh>
 #include <corecel/io/Logger.hh>
 
+using TMI = celeritas::TrackingManagerIntegration;
+
 namespace
 {
@@ -207,9 +212,15 @@ int main()
     run_manager->SetUserInitialization(new DetectorConstruction{});
 
+    auto& tmi = TMI::Instance();
+
     // Use FTFP_BERT, but use Celeritas tracking for e-/e+/g
     auto* physics_list = new FTFP_BERT{/* verbosity = */ 0};
+    physics_list->RegisterPhysics(
+        new celeritas::TrackingManagerConstructor(&tmi));
     run_manager->SetUserInitialization(physics_list);
     run_manager->SetUserInitialization(new ActionInitialization());
 
+    tmi.SetOptions(MakeOptions());
+
     run_manager->Initialize();
 
